packages/grafana-ui/src/components/Pagination/Pagination.story.tsx

```diff
@@ -1,22 +1,47 @@
 import React, { useState } from 'react';
-import { number } from '@storybook/addon-knobs';
+import { Story } from '@storybook/react';
 import { withCenteredStory } from '../../utils/storybook/withCenteredStory';
 import { Pagination } from '@grafana/ui';
 import mdx from './Pagination.mdx';
-
-export const WithPages = () => {
-  const [page, setPage] = useState(1);
-  const numberOfPages = number('Number of pages', 5);
-  return <Pagination numberOfPages={numberOfPages} currentPage={page} onNavigate={setPage} />;
-};
+import { Props } from './Pagination';
 
 export default {
   title: 'Buttons/Pagination',
-  component: WithPages,
+  component: Pagination,
   decorators: [withCenteredStory],
   parameters: {
     docs: {
       page: mdx,
     },
+    knobs: {
+      disable: true,
+    },
+    controls: {
+      exclude: ['currentPage', 'onNavigate'],
```

Comment:
Curious why we're excluding `currentPage`? It could be used when users want to see how pagination looks at a specific page for ex.

Reply:
I think it's easier to access the current page from the story itself instead of explicitly doing so via the control.

Reply:
We could add a check that the current page doesn't exceed the number of total pages, but that's probably not gonna be used often enough, so can be left as it is.